maven-failsafe-plugin/src/main/java/org/apache/maven/plugin/failsafe/VerifyMojo.java

```diff
@@ -236,6 +243,11 @@ protected boolean verifyParameters()
             return false;
         }
 
+        if ( failOnFlakeCount < 0 )
```

Comment:
This must be in the abstract class where are all the warning check method calls made in the section:
```
        else
        {
            ensureEnableProcessChecker();
            convertDeprecatedForkMode();
            ensureWorkingDirectoryExists();
            ensureParallelRunningCompatibility();
            ensureThreadCountWithPerThread();
            warnIfUselessUseSystemClassLoaderParameter();
            warnIfDefunctGroupsCombinations();
            warnIfRerunClashes();
            warnIfWrongShutdownValue();
            warnIfNotApplicableSkipAfterFailureCount();
            warnIfIllegalTempDir();
            warnIfForkCountIsZero();
            printDefaultSeedIfNecessary();
        }
```